R/assertions.R

```diff
@@ -1480,25 +1485,24 @@ assert_expr_list <- function(arg, # nolint
     ~ is_call(.x) || is_expression(.x)
   )
   invalidargs <- arg[invalids]
+  index <- if_else(names(invalidargs) == "", as.character(which(invalids)),
+    paste0('"', names(invalidargs), '"')
+  )
 
   if (any(invalids)) {
-    argname <- arg_name(substitute(arg))
-    abort(
-      paste0(
-        "All elements of `",
-        argname,
-        "` must be an expression.\n",
-        paste0(
-          argname,
-          "[[",
-          if_else(names(invalidargs) == "", as.character(which(invalids)), names(invalidargs)),
-          "]] = ",
-          map_chr(invalidargs, expr_label),
-          " is of type ",
-          map_chr(invalidargs, typeof),
-          collapse = "\n"
-        )
-      )
+    cli_abort(
+      message = message %||%
+        c("All elements of {.arg {arg_name}} must be an expression.",
```

Comment:
```suggestion
        c("All elements of {.arg {arg_name}} argument must be an expression.",
```
Does including the word argument help? 
![image](https://github.com/pharmaverse/admiraldev/assets/10111024/ade8d927-b9f3-4cc6-bc27-85756de4b73b)

versus 

![image](https://github.com/pharmaverse/admiraldev/assets/10111024/f8ee24c2-4568-4b0e-afe2-8cb20b137999)

Reply:
Since we're testing what users pass as arguments, I like the default argument text (which can be changed) 🤷🏼